openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -312,6 +318,42 @@ def list(self, request):
 
         return Response(account_settings)
 
+    def search_emails(self, request):
+        """
+        POST /api/user/v1/accounts/search_emails
+        Content Type: "application/merge-patch+json"
+        {
+            "emails": ["edx@example.com", "staff@example.com"]
+        }
+        Response:
+        [
+            {
+                "username": "edx",
+                "email": "edx@example.com",
+                "id": 3,
+            },
+            {
+                "username": "staff",
+                "email": "staff@example.com",
+                "id": 8,
+            }
+        ]
+        """
+        try:
+            user_emails = request.data['emails']
+        except KeyError as err:
+            mesg = f'{err} field is required'
```

Comment:
can we please rename it something like error_msg?

Reply:
sure